src/package/VSIXProject/testplatform.config

```diff
@@ -2,7 +2,9 @@
 <!-- This file determines whether to use TestPlatform v1 or TestPlatform v2 for tests execution. -->
 <configuration>
   <appSettings>
-    <!-- Setting feature.net40 to "true" will execute full clr tests through TPV2. -->
+    <!-- Setting feature.net35 to "true" will execute full clr tests targeting framework <=v3.5 through TPV2. -->
```

Comment:
How will it execute?

Reply:
Tests targeting v3.5 tests will be executed with v4.0 Runtime in compat mode.